gemfirexd/shared/src/main/java/io/snappydata/thrift/common/OptimizedElementArray.java

```diff
@@ -96,31 +96,32 @@
  *     header      body
  * </pre>
  */
+@SuppressWarnings("WeakerAccess")
```

Comment:
nit: we can change access qualifiers instead of suppressing warning.

Reply:
skipping for now. Will make changes separately